### PR TITLE
feat: sync UI with lottie.github.io

### PR DESCRIPTION
**Pull request by Aidosmf**
https://lottiefiles.github.io/lottie-spec/

### CODE REVIEW

**Comment by mbasaglia** [comment]
The arrow in sub-menus isn't very visible, also sub-menus require a click to open, they should just open on hover

Screenshot_20240130_163415

**Comment by mbasaglia** [comment]
This page has metadata to hide the TOC but it's still being rendered

Screenshot_20240130_163545

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Build Pages
 on:
     push:
         branches:
-            - main
-            - docs-setup
+            # - main
+            - mf/ui-updates
```
Comment (mbasaglia) [inline .github/workflows/pages.yml:6]: This should go back to main before merging

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,7 @@
 docs/lottie.schema.json
 site/
+tools/schema_tools
```
Comment (mbasaglia) [inline .gitignore:3]: This should not be in gitignore

File: mkdocs.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ site_description: Documentation for the Lottie animation format
 extra:
     lottie_player_version: 5.12.2
 theme:
-    name: cinder
```
Comment (mbasaglia) [inline mkdocs.yml:6]: If cinder is removed, it should be removed from requirements.txt as well

**Review by mbasaglia — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 3 above)
Comment (Aidosmf) [reply mkdocs.yml]: brought back the theme with the updated styles

**Comment by Aidosmf** [comment]
@mbasaglia thanks for the review. I brought back the cinder theme with the stylings and I have applied all the suggestions.

**Comment by mbasaglia** [comment]
> @mbasaglia thanks for the review. I brought back the cinder theme with the new stylings and I have applied all the suggestions.

Does the CSS stuff still work fine with it?

**Comment by Aidosmf** [comment]
> Does the CSS stuff still work fine with it?

I adjusted css and tested it, seems fine, unless you'd find anything

**Review by mbasaglia — APPROVED** [review]

**Review by b-wils — APPROVED** [review]

**Review by kudanai — APPROVED** [review]

**Pull request merged**